src/main/scala/nl.knaw.dans.easy.dd2d/Configuration.scala

```diff
@@ -46,6 +48,13 @@ object Configuration {
       load((cfgPath / "application.properties").toJava)
     }
 
+    val narcisClassificationPath = Seq(
+      root / "opt" / "dans.knaw.nl" / "dd-dans-deposit-to-dataverse" / "install" / "narcis_classification.xml",
+      home / "install" / "narcis_classification.xml")
+      .find(_.exists)
+      .getOrElse { throw new IllegalStateException("No Narcis Classification file found") }.canonicalPath
```

Comment:
Is it necessary for the whole program to exit when this resource is not available? It is only used in one place and for the mapping of one field.

Reply:
@ChrisMuller80 
I'd say "yes, let it fail", for two reasons:

* This field is used in every dataset.
* Even if it were not, it is better to avoid starting applications in a crippled mode. This just generates ambiguity. Better to start only if all resources are available and if not: fail fast, which is what this code does.